<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: '#282b30';
}
QWidget {
background-color: '#282b30';
color: '#d7ddde';
}
QLineEdit{
border-radius: 2px;
background-color: '#484b52';
padding: 4px;
}
QComboBox {
border-radius: 2px;
background-color: '#484b52';
padding: 4px;
}
QPushButton{
border-radius: 2px;
background-color: '#7289da';
color: white;
font-weight: bold;
padding: 6px;
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="firstFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QGroupBox" name="videoGroup">
         <property name="title">
          <string>Vídeo</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="videoDir">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diretório do vídeo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="findVideoDir">
               <property name="text">
                <string>Procurar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QComboBox" name="chooseVideoFormat">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formato do vídeo final&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <item>
                <property name="text">
                 <string>MP4</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="secondFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QGroupBox" name="audioGroup">
         <property name="title">
          <string>Áudio</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="minimumSize">
             <size>
              <width>550</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QComboBox" name="chooseAudioFormat">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formato do áudio final&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <item>
                <property name="text">
                 <string>AAC</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="audioQuality">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bitrate do áudio final&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>128k</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QRadioButton" name="withAudio">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>C/Áudio</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="withoutAudio">
               <property name="text">
                <string>S/Áudio</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="thirdFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QGroupBox" name="subGroup">
         <property name="title">
          <string>Legenda</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <property name="minimumSize">
             <size>
              <width>480</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>480</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLineEdit" name="subDir">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diretório da legenda&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="findSubDir">
               <property name="text">
                <string>Procurar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QComboBox" name="chooseSubFormat">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Formato da legenda final&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <item>
                <property name="text">
                 <string>SRT</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ASS</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QRadioButton" name="withSub">
               <property name="text">
                <string>C/Legenda</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="withoutSub">
               <property name="text">
                <string>S/Legenda</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fourthFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QGroupBox" name="generalGroup">
         <property name="title">
          <string>Geral</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="crfNum">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CRF&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="whatsThis">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>18</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="preset">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Preset de velocidade&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <item>
                <property name="text">
                 <string>ultrafast</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>superfast</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>veryfast</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>faster</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>fast</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>medium</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>slow</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>slower</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>veryslow</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="videoWidth">
               <property name="text">
                <string>1920</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="videoHeight">
               <property name="text">
                <string>1080</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="timeStart">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tempo de início para conversão&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>00:00:00.000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="timeEnd">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tempo de fim para conversão&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>00:00:00.000</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fifithFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <item>
        <widget class="QGroupBox" name="outputGroup">
         <property name="title">
          <string>Saída</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QLineEdit" name="outputDir">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diretório de saída&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="findOutputDir">
               <property name="text">
                <string>Procurar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_10" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QPushButton" name="execute">
               <property name="text">
                <string>Executar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>withoutAudio</sender>
   <signal>clicked()</signal>
   <receiver>chooseAudioFormat</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withoutAudio</sender>
   <signal>clicked()</signal>
   <receiver>audioQuality</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withAudio</sender>
   <signal>clicked()</signal>
   <receiver>chooseAudioFormat</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withAudio</sender>
   <signal>clicked()</signal>
   <receiver>audioQuality</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withoutSub</sender>
   <signal>clicked()</signal>
   <receiver>subDir</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withoutSub</sender>
   <signal>clicked()</signal>
   <receiver>findSubDir</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withoutSub</sender>
   <signal>clicked()</signal>
   <receiver>chooseSubFormat</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withSub</sender>
   <signal>clicked()</signal>
   <receiver>subDir</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withSub</sender>
   <signal>clicked()</signal>
   <receiver>findSubDir</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withSub</sender>
   <signal>clicked()</signal>
   <receiver>chooseSubFormat</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
